CMSgov/design-system
# [NO-TICKET] Fix the release notes script using older tags for version numbers

## Summary

The problem with the command `git tag -l` was that it sorts tags by string value. Our logic relied on the sort order being numeric. After version 10, the most recent version no longer appeared at the end of the list. 9.0.2 is the last version in the list, because 10, 11, and 12 come before 9 in alphabetical string-sort order. It has now been changed to order by most recently created. Documented assumptions as well.

## How to test

Run `yarn release:notes` but don't publish them. Check that the version numbers in any headings created are the most recently tagged version numbers. Note that this assumption only serves us in situations where we are creating the release notes for the release we just created, not creating release notes for an older release after other releases have happened.
